documentation/release_notes.markdown

```diff
@@ -12,6 +12,20 @@ canonical: "/puppetserver/latest/release_notes.html"
 
 For release notes on versions of Puppet Server prior to Puppet Server 2.5, see [docs.puppet.com](https://docs.puppet.com/puppetserver/2.4/release_notes.html).
 
+## Puppet Server 2.8.0
+
+Released August 31, 2017
```

Comment:
Is this still the expected release date?

Reply:
Yeah, that's my understanding